cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -8,30 +8,31 @@ category "Cost"
 tenancy "single"
 default_frequency "daily"
 info(
-  version: "2.3",
+  version: "2.4.0",
   provider: "Flexera",
   service: "Cloud Cost Optimization",
   policy_set: "Cloud Cost Optimization"
 )
 
-###############################################################################
-# Permissions
-###############################################################################
-
-
 ###############################################################################
 # Parameters
 ###############################################################################
 
 parameter "param_email" do
   type "list"
+  category "Policy Settings"
   label "Email addresses of the recipients you wish to notify"
+  description "A list of email addresses to notify."
+  # No default value, user input required
```

Comment:
Email parameter should always have a default value of [] unless, for some reason, you expect that nobody would ever use the policy just to see the incident within the Flexera One platform and would *only* want the incident to be emailed.